### DISCUSSION TITLE
SB5.1 - OCI - Fresh installation 5.0/5.1: Deploy/Lern/LernFlinkJobs, is it a doc bug?

### DISCUSSION THREAD

**Discussion by heungheung** [Sunbird-Ed/Community#324] [Documentation] [answered]
In the update to 5.0.0 doc of Sunbird-Ed
  https://ed.sunbird.org/use/updating-sunbird-releases/ed-portal/4.10.1-4.10.2-4.10.3-and-5.0
it mentioned to build and deploy this new Lern BB components

Build/Lern/LernDataProducts | release-5.0.0_RC3 | Deploy/Lern/LernDataProducts | release-5.0.0_RC3 | New Lern BB jobs
-- | -- | -- | -- | --
Build/Lern/LernFlinkJobs | release-5.0.0_RC4 | Deploy/Lern/LernFlinkJobs | release-5.0.0_RC4 | New Lern BB jobs

However, for new/fresh installation
  https://ed.sunbird.org/use/installing-ed-platform/release-5.0.0/build
  https://ed.sunbird.org/use/installing-ed-platform/release-5.1.0-draft/build
They only talked about "Build". There is no where in the Deploy section to talk about running the corresponding Deploy pipeline job to get the Lern BB component deployed. 

Is this a doc bug?

**Comment by beepdot** [comment, marked as answer]
@saiakhil46 Looks like we have missed to add the deploy section to docs. Please add them.
@santhosh-tg The inventory structure for these jobs is missing in private repo templates. Please raise a PR for the same.

**Reply by heungheung** [reply]
Thanks for confirmation. Marking as answer.
